### PR TITLE
PLAT-853: Created zen-lozenge component

### PR DESCRIPTION
**Pull request by f-jauregui**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-853)

Captura de Pantalla 2021-04-19 a la(s) 23 06 29

#### Description
As per JIRA ticket

#### ChangeLOG

[Changelog](https://zen-ui.zengrc.com/?path=/docs/changelog--page)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

File: src/components/zen-lozenge/zen-lozenge.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
+@import '../scss/component-helpers.scss';
+
+:host {
+  display: flex;
```
Comment (homer0) [inline src/components/zen-lozenge/zen-lozenge.scss:4]: Almost sure that when we discussed the issue with the tabs width, someone said something about our components being `block`... not sure, maybe Jan (too late for mentions) will have more context here.

File: src/components/zen-lozenge/zen-lozenge.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      <sb-zen-space spacing="xl">
+        <sb-zen-lozenge></sb-zen-lozenge>
+        <sb-zen-lozenge text-variant="capitalize"></sb-zen-lozenge>
+        <sb-zen-lozenge size="large"></sb-zen-lozenge>
+        <sb-zen-lozenge size="large" text-variant="capitalize"></sb-zen-lozenge>
+      </sb-zen-space>
+      <sb-zen-space spacing="xl">
+        <sb-zen-lozenge variant="dark-grey"></sb-zen-lozenge>
+        <sb-zen-lozenge variant="dark-grey" text-variant="capitalize"></sb-zen-lozenge>
+        <sb-zen-lozenge variant="dark-grey" size="large"></sb-zen-lozenge>
```
Comment (homer0) [inline src/components/zen-lozenge/zen-lozenge.stories.mdx:27]: Shouldn't the `size` be `lg`? I see that the stylesheet and the component only support `default` and `lg`, but only with `large` is that you get the "large lozenge" :P.

**Review by homer0 — APPROVED** [review]
🤘... two small questions

**Review by chavlji — APPROVED** [review]
Looks good. Just a few minor tweaks...

File: src/components/helpers/types.ts [hunk 3]
```diff
@@ -37,6 +37,23 @@ export type AvatarDetailVariant = 'basic' | 'basic-lg' | 'detailed';
...
+  | 'dark-purple'
+  | 'light-blue'
+  | 'dark-blue'
+  | 'green'
+  | 'red'
+  | 'dark-blue-ghost';
+
+export type LozengeSize = 'default' | 'lg';
+
+export type LozengeTextVariant = 'uppercase' | 'capitalize';
```
[the remark this answers is not in the document]
Comment (homer0) [reply src/components/helpers/types.ts:55]: Based on the Figma, nop:

> Use all caps when indicating statuses.  Other use cases (i.e. tags on program setup step one tiles) can use lower case with first letter capitalized).

File: src/components/zen-lozenge/zen-lozenge.stories.mdx (hunk 2 above)
Comment (f-jauregui) [reply src/components/zen-lozenge/zen-lozenge.stories.mdx:27]: True.

File: src/components/zen-lozenge/zen-lozenge.scss (hunk 1 above)
Comment (f-jauregui) [reply src/components/zen-lozenge/zen-lozenge.scss:4]: We will go with inline-flex as @chavlji mentioned.

File: src/components/helpers/types.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (f-jauregui) [reply src/components/helpers/types.ts:55]: Nope, we only have uppercase by default and capitalize as mentioned [here](https://www.figma.com/file/rWXEQiJcBXbkMY8V4NSM1d/ZenGRC-Component-Library?node-id=1096%3A233): 

Captura de Pantalla 2021-04-20 a la(s) 14 35 47

**Comment by mihaelpalfi** [comment]
What is a lozenge??

**Comment by homer0** [comment]
@mihaelpalfi

> a small medicinal tablet, originally in the shape of a lozenge, taken for sore throats and dissolved in the mouth.
>
> _- [Google](https://www.google.com/search?q=lozenge+definition&oq=lozenge+definition&aqs=edge..69i57j0l4j0i22i30l2.3391j0j1&sourceid=chrome&ie=UTF-8)_

So, literally, a pill xD

**Pull request merged**